Lib/pathlib.py

```diff
@@ -194,25 +189,23 @@ def resolve(self, path, strict=False):
         s = str(path)
         if not s:
             return os.getcwd()
```

Comment:
```suggestion
            s = os.getcwd()
```

Reply:
I wonder if this should belong in a separate change as well since this condition is not related to any change proposed here.